src/frontends/tensorflow/tests/convert_saved_model.cpp

```diff
@@ -114,3 +114,13 @@ TEST_F(TransformationTestsF, SavedModelMultipleTensorNames) {
         model_ref = make_shared<Model>(OutputVector{result}, ParameterVector{x});
     }
 }
+
+TEST_F(TransformationTestsF, SavedModelBroadcastIssue) {
+    { model = convert_model("saved_model_broadcast_issue"); }
+    {
+        // create a reference graph
+        auto x = make_shared<Constant>(element::i64, Shape{2,2}, vector<int64_t>{0,0,0,0});
+
+        model_ref = make_shared<Model>(OutputVector{x}, ParameterVector{});
```

Comment:
`ConstToResultRemover` does not remove `Const->Result` sub-graph?

Reply:
Fortunately (?) - no. Because at this point graph isn't Const->Result (it'll be Const->Result after TransposeSinking). Otherwise, I think, it produces wrong model without nodes.